addons/website_slides/static/src/js/public/components/slide_share_dialog/email_sharing_input.js

```diff
@@ -0,0 +1,59 @@
+/** @odoo-module **/
+
+import { rpc } from "@web/core/network/rpc";
+import { session } from "@web/session";
+import { useService } from "@web/core/utils/hooks";
+import { _t } from "@web/core/l10n/translation";
+
+import { Component, useRef, useState } from "@odoo/owl";
+
+export class EmailSharingInput extends Component {
+    static template = "website_slides.EmailSharingInput";
+    static props = {
+        id: { type: Number },
+        isChannel: { type: Boolean, optional: true },
+        isFullscreen: { type: Boolean, optional: true },
+        category: { type: String, optional: true },
+    };
+
+    setup() {
+        this.notification = useService("notification");
+        this.input = useRef("input");
+        this.isWebsiteUser = session.is_website_user;
+        this.state = useState({
+            isDone: false,
+            isInvalid: false,
+        });
+    }
+
+    onKeyPress(event) {
+        if (event.keyCode === 13) {
```

Comment:
I see that in the code `ev.key === "Enter"`, can you check that it work ?
(it will be more explicit than 13)